app/models/map.rb

```diff
@@ -78,7 +78,6 @@ def generate_identifier(token)
   end
 
   def scale_coordinates
-    maybe_analyze_image
```

Comment:
This was messing up updating the map for the demo, and I removed it and tested it and it didn't seem to have an effect.